CHANGES_NEXT_RELEASE

```diff
@@ -15,6 +15,9 @@
 - Fix: bug when tenants are over 44 bytes long (#2811)
 - Add: Support for null values in string filters (#2359)
 - Fix: Partial sanity check for custon url, only if the url contains no replacements using ${xxx} (#2280, #2279)
+- Fix: if entity::type is given but as an empty string, an error is returned (#1985)
+- Fix: admin requests with errors now return '400 Bad Request', not '200 OK'
```

Comment:
Sure, an example may make it easier to understand.
No, no issue here, as far as I know, see PR description:
```
admin requests with errors now return '400 Bad Request', not '200 OK' (no issue, I think)
```